queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth>= '2016-01-01' AND date_of_birth<='2019-12-31';
+SELECT * FROM animals WHERE neutered = 'Yes' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, date_of_birth FROM animals WHERE weight_kg > 10.5;
```

Comment:
This query should list the name and escape attempts of animals and not the date of birth. Kindly fix this issue.